coolify/setup.sh

```
@@ -52,15 +51,60 @@ else
+# NOTE: This is required on Linux/macOS. Windows Docker Desktop handles permissions automatically.
+if [ ! -d "logs/postgres" ]; then
+    mkdir -p logs/postgres
+    print_success "Created logs/postgres/ directory"
 else
-    print_success "Found config/ directory"
+    print_warning "logs/postgres/ directory already exists"
 fi
 
+# Try to set correct ownership (UID 999 = postgres user in container)
+if command -v sudo &> /dev/null && [ "$OSTYPE" != "msys" ] && [ "$OSTYPE" != "win32" ]; then
+    sudo chown -R 999:999 logs/postgres 2>/dev/null && \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)